composer.json

```diff
@@ -148,7 +148,7 @@
         "symfony/webpack-encore-bundle": "^1.7",
         "symfony/workflow": "^5.4",
         "tracy/tracy": "^2.4.13",
-        "twig/twig": "^2.12",
+        "twig/twig": "^v2.15.4",
```

Comment:
One more thing, there should not be "v" at the begining

Reply:
ok